VRSampleApp/app/build.gradle

```diff
@@ -38,6 +38,7 @@ android {
 task copyAssetsTask(type: Copy) {
     from new File(vendorDir, 'Ooyala/OoyalaSkinSDK-Android/index.android.jsbundle')
     from new File(vendorDir, 'Ooyala/OoyalaSkinSDK-Android/skin-config/skin.json')
+    from new File(vendorDir, 'Ooyala/OoyalaSkinSDK-Android/data.json')
```

Comment:
We removed data.json from OoyalaSkinSDK-Android in the vendor folder. It makes more sense to have it part of the app instead of bundling it with the SDK.

You can remove this line.

Reply:
Done